packages/storybook/stories/MeetingComposite/snippets/Meeting.snippet.tsx

```diff
@@ -0,0 +1,80 @@
+import { GroupCallLocator, TeamsMeetingLinkLocator } from '@azure/communication-calling';
+import { AzureCommunicationTokenCredential, CommunicationUserIdentifier } from '@azure/communication-common';
+import { Theme, PartialTheme } from '@fluentui/react';
+import {
+  CallAdapter,
+  ChatAdapter,
+  MeetingComposite,
+  createAzureCommunicationCallAdapter,
+  createAzureCommunicationChatAdapter
+} from '@internal/react-composites';
+import React, { useState, useEffect, useMemo } from 'react';
+
+export type MeetingExampleProps = {
+  userId: CommunicationUserIdentifier;
+  token: string;
+  displayName: string;
+  endpointUrl: string;
+  locator: GroupCallLocator | TeamsMeetingLinkLocator;
+  threadId: string;
+  fluentTheme?: PartialTheme | Theme;
+  callInvitationURL?: string;
+};
+
+export const MeetingExperience = (props: MeetingExampleProps): JSX.Element => {
+  const [callAdapter, setCallAdapter] = useState<CallAdapter>();
+  const [chatAdapter, setChatAdapter] = useState<ChatAdapter>();
+
+  const credential = useMemo(() => {
+    try {
+      return new AzureCommunicationTokenCredential(props.token);
+    } catch {
+      console.error('Failed to construct token credential');
+      return undefined;
+    }
+  }, [props.token]);
+
+  useEffect(() => {
+    if (
+      props &&
+      credential &&
+      props.locator &&
+      props.displayName &&
+      props.threadId &&
+      props.userId &&
+      props.endpointUrl
+    ) {
+      const createAdapters = async (): Promise<void> => {
+        setCallAdapter(
+          await createAzureCommunicationCallAdapter(
+            { kind: 'communicationUser', communicationUserId: props.userId.communicationUserId },
+            props.displayName,
+            credential,
+            props.locator
+          )
+        );
+
+        setChatAdapter(
+          await createAzureCommunicationChatAdapter(
+            props.endpointUrl,
+            { kind: 'communicationUser', communicationUserId: props.userId.communicationUserId },
+            props.displayName,
+            credential,
+            props.threadId
+          )
+        );
+      };
+      createAdapters();
+    }
+  }, [credential, props]);
+
+  if (callAdapter && chatAdapter) {
+    return <MeetingComposite callAdapter={callAdapter} chatAdapter={chatAdapter} fluentTheme={props.fluentTheme} />;
+  }
+
+  if (credential === undefined) {
+    return <>Failed to construct credential. Provided token is malformed.</>;
```

Comment:
Is that a standard verbatim used at other places?

Reply:
Yeah used in the other composite samples (and is not an imported constant as it doesn't lend itself well to copy-pastable snippets)